echopype/consolidate/api.py

```diff
@@ -240,19 +255,20 @@ def add_splitbeam_angle(
     storage_options: dict, default={}
         Any additional parameters for the storage backend, corresponding to the
         path provided for ``source_Sv``
-    return_dataset: bool, default=True
-        If ``True``, ``source_Sv`` with split-beam angles added will be returned.
-        ``return_dataset=False`` is useful when ``source_Sv`` is a path and
+    to_disk: bool, default=True
+        If ``True``, ``to_disk`` with split-beam angles added will be returned.
+        ``to_disk=False`` is useful when ``source_Sv`` is a path and
         users only want to write the split-beam angle data to this path.
 
     Returns
     -------
     xr.Dataset or None
-        If ``return_dataset=False``, nothing will be returned.
-        If ``return_dataset=True``, either the input dataset ``source_Sv``
+        If ``to_disk=False``, nothing will be returned.
+        If ``to_disk=True``, either the input dataset ``source_Sv``
         or a lazy-loaded Dataset (from the path ``source_Sv``)
         with split-beam angles added will be returned.
```

Comment:
I think the True and False conditions here are reverse.

> to_disk=True means that the angle will be saved to disk via mode="a" and the source_Sv dataset lazy-loaded back and returned, and to_disk=False means that the angle will just be added to source_Sv and returned but not saved to disk.

Reply:
Updated!